Cargo.toml

```diff
@@ -114,7 +114,7 @@ paste = { version = "1.0.12", default-features = false }
 percent-encoding = { version = "2.2.0", default-features = false }
 pretty_assertions = { version = "1.3.0", default-features = false }
 proc-macro2 = { version = "1.0.56", default-features = false }
-pyo3 = { version = "0.17.3", default-features = false }
+pyo3 = { version = "0.19.1", default-features = false }
```

Comment:
FYI: A PR is already openned for that #4853

Reply:
👍 I didn't know